tools/syz-reprolist/reprolist.go

```diff
@@ -28,10 +32,111 @@ var (
 	flagOutputDir    = flag.String("output", "repros", "output dir")
 	flagSyzkallerDir = flag.String("syzkaller", ".", "syzkaller dir")
 	flagOS           = flag.String("os", runtime.GOOS, "target OS")
+	flagNamespace    = flag.String("namespace", "", "target namespace")
+	flagToken        = flag.String("token", "", "gcp bearer token to disable throttling (contact syzbot first)\n"+
+		"usage example: ./tools/syz-reprolist -namespace upstream -token $(gcloud auth pring-access-token)")
 )
 
+func getJSONBody(url string) ([]byte, error) {
```

Comment:
Please place all of this code below. This is not C/C++ that forces inverted placement of things.